ext/rugged/rugged_commit.c

```diff
@@ -662,11 +693,77 @@ static VALUE rb_git_commit_extract_signature(int argc, VALUE *argv, VALUE self)
 	return ret;
 }
 
+/*
+ *  call-seq:
+ *    Commit.create_buffer(repository, data = {}) -> commit
```

Comment:
`#compose` seems like a weird name... *we're* not composing anything here... maybe `#create_as_string()` or something with `in_memorry` in the name?